.azure-pipelines/ci-build.yml

```diff
@@ -1,869 +1,163 @@
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
-
-name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
-
 trigger:
+  branches:
+    include:
+      - main
+      - task/add-new-acr-pipeline
   paths:
-    exclude:
-      [
-        "abstractions/**",
-        "authentication/**",
-        "serialization/**",
-        "http/**",
-        "**.md",
-        ".vscode/**",
-        "**.svg",
-      ]
+    include:
+      - src/**
+      - .azure-pipelines/**
   tags:
     include:
-      - "v*"
-    exclude:
-      - "*preview*" # exclude preview tags so we don't publish twice
+      - v*
+
 schedules:
-  - cron: "0 18 * * 4" # 18:00 UTC every Thursday ~ 14:00 EST every Thursday
-    displayName: Weekly preview
-    branches:
-      include:
-        - main
-    always: true
+- cron: "0 18 * * 4"  # 18:00 UTC every Thursday ~ 14:00 EST every Thursday
+  displayName: Weekly build
+  branches:
+    include:
+      - main
+  always: true
 
 variables:
-  buildPlatform: "Any CPU"
-  buildConfiguration: "Release"
-  ProductBinPath: '$(Build.SourcesDirectory)\src\kiota\bin\$(BuildConfiguration)'
-
-parameters:
-  - name: previewBranch
-    type: string
-    default: "refs/heads/main"
-  - name: distributions
-    type: object
-    default:
-      - architecture: "win-x86"
```

Comment:
assuming this removal is just for testing purposes and will be reverted back to original